app/frontend/components/pages/login/mnemonicAuth.tsx

```diff
@@ -8,15 +8,17 @@ import tooltip from '../../common/tooltip'
 import Alert from '../../common/alert'
 import sanitizeMnemonic from '../../../helpers/sanitizeMnemonic'
 import {ADALITE_CONFIG} from '../../../config'
+import {mnemonicValidator} from '../../../helpers/validators'
+
 const {ADALITE_DEMO_WALLET_MNEMONIC} = ADALITE_CONFIG
 
 interface Props {
   mnemonicInputValue: string
   mnemonicValidationError: any
   updateMnemonic: (e: any) => void
-  checkForMnemonicValidationError: () => void
+  updateMnemonicValidationError: () => void
+  //
```

Comment:
did you forget this?

Reply:
same as above, make sure prettier does not collapse things